packages/ethers5/src/client.ts

```diff
@@ -236,6 +236,7 @@ export class Web3Modal extends Web3ModalScaffold {
     this.options = options
 
     this.metadata = ethersConfig.metadata
+    this.setMetadata(this.metadata)
```

Comment:
Metadata should be set as any other option and then passed to options controller inside scaffold.initControllers method. Why do we need to create and use separate method for it here?